PSix/ParseUserSession.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  CurrentUser.swift
+//  PSix
+//
+//  Created by Turzion, Avihu on 4/14/15.
+//  Copyright (c) 2015 PayPal. All rights reserved.
+//
+
+import Foundation
+import Parse
+
+class ParseUserSession {
+    
+    private static let FB_READ_PERMISSIONS = ["public_profile", "user_events"]
+    
+    static private var currentLoggedInUser: User? = nil
+    
+    static var currentUser: User? {
+        get {
+            if currentLoggedInUser == nil && PFUser.currentUser() != nil {
+                currentLoggedInUser = User(withoutDataWithObjectId: PFUser.currentUser()?.objectId)
+                currentLoggedInUser?.fetch()
+            } else if currentLoggedInUser == nil && PFUser.currentUser() == nil {
+                return nil
+            }
+            return currentLoggedInUser
+        }
```

Comment:
I had a design issue here that caused this mess which I wasn't sure how to solve before the PR. The thing is that I wanted to add more attributes to the Parse user (specifically the Facebook ID), and wanted the model to reflect that. I wanted to extend that to all users that will later RSVP to the events just to realize (as I'm going over this just now) that the other users that RSVP are most likely not PSix users. So I think I'll ditch this and go back to using the regular `PFUser` class.